README.md

```diff
@@ -1 +1,13 @@
-# hw01
\ No newline at end of file
+
+
+### My biography
+
+My name is **Guillermo Guevara** and I am second year MPP at Harris. Before coming to Harris, I held several positions at the Mexican Federal Government.I am interested in:
+
+- Policing
```

Comment:
Great job with the bullet points. Although it is different than the way I wrote the code out, it came out well!